src/resources/views/crud/columns/password.blade.php

```diff
@@ -0,0 +1,22 @@
+@php

```

Comment:
I am not totally convinced that we should be doing this column. 
- What is the point of showing `**********` in a table ?
- We limit it to 6, so it will not even be accurate, most of the time would be `******...` 
- If dev make the limit `999` it will be a security issue because will be exposing the length of passwords. 

In the end this column introduces a very very bad practice that I think we should not encourage this behavior as it can be very dangerous. 

@tabacitu

Reply:
So should we ignore this column type?

Reply:
ping @tabacitu

Reply:
The purpose is for _each field to have a corresponding column type_. This is in preparation for fields and columns being merged (in the next version). At that point, when we rewrite the fields to objects, all fields HAVE TO have a read-only-state. Otherwise it'd fall back to the default `text` behaviour and just output the value.

In addition, this would provide a better default for when people generate stuff. Say you generate a `UserCrudController`, which would have a `password` db column too, right? Right now when you do you see the hash in the table. Ideally, by next year, when you generate that, it'd use the `password` column type and you'd see it ***** so it's _safer by default_.

**I say let's do it.** Most people won't use it... I agree, but that's fine. It would add consistency, and that's reason enough. No point in "_adding columns for all field types_" if we don't do that for `password`, right?

----

Heads-up @munjaldevelopment - why doesn't this column type just do a little processing (set the value to ***), then include the text column type? That's the preferred way to do columns, so we gotta have a good reason NOT TO do that.

Reply:
If I call, include text. It would add ... (and so on) at the end. Is that fine? Otherwise I need to change that part to remove '...' only..

Reply:
Hmmm I don't think it would add "..." at the end, if before we load the text column we override the value to be `******` - it is unlikely that anyone will ever set the limit to lower than 6 characters.

Reply:
applied changes as discussed.

Reply:
Awesome, thank you @munjaldevelopment 🙏

Reply:
<3